exercises/practice/hamming/.docs/instructions.append.md

```diff
@@ -0,0 +1,14 @@
+# Implementation notes: exceptions
+
+This exercise expects the code to raise exceptions in certain circumstances.
+Exceptions are usually raised when an operation cannot be completed for some
+reason and there is no value that can be returned which would be correct.
```

Comment:
I think you meant: and there is no applicable value to return?

"which would be correct" is amtrigous. Either the act of no value returning is correct, or returning a value would be correct, or a correct value cannot be returned.

Reply:
Operation cannot be performed. Function usually returns a value. Any value returned would be incorrect. There exists no value where returning that value would be correct.

Maybe,
> Exceptions are usually raised when an operation cannot be completed for some reason and therefore there is no value to return. Any value returned would be an incorrect value.

Reply:
Yeah much better!